train/split_video.py

```diff
@@ -51,6 +51,8 @@
     original_img = img.copy()
     if args.no_display:
         key = RIGHT_KEY  # Next frame
+        if current_idx == n_frames - 1:
```

Comment:
if current_idx >= n_frames - 1 ?

Reply:
It is equivalent, no ? because the current_idx cannot be > n_frames
line 65 of the same file: `current_idx = np.clip(current_idx, 0, n_frames - 1)`